crates/transaction-pool/src/noop.rs

```diff
@@ -337,12 +337,14 @@ impl<T: EthPoolTransaction> TransactionValidator for MockTransactionValidator<T>
         TransactionValidationOutcome::Valid {
             balance: U256::MAX,
             state_nonce: 0,
+            bytecode_hash: None,
```

Comment:
I think we should consider grouping all of this into a new struct, but not in this pr

Reply:
i will do it in next pr